testplan/web_ui/testing/src/Toolbar/Toolbar.js

```diff
@@ -503,10 +504,24 @@ const getToggledButtonStyle = (status) => {
  * Get the metadata from the report and render it as a table.
  */
 const getInfoTable = (report) => {
+  
+
   if (!report || !report.information) {
     return "No information to display.";
   }
   const infoList = report.information.map((item, i) => {
+    const linkifyIgnore = ['user', 'command_line_string', 'python_version', 'hostname', 'start', 'end']
+    if (!linkifyIgnore.includes(item[0])) {
+      return (
+        <tr key={i}>
+          <td className={css(styles.infoTableKey)}>{item[0]}</td>
+          <td className={css(styles.infoTableValue)}><div dangerouslySetInnerHTML={{__html:linkifyUrls(
```

Comment:
There is too many duplication here with the ignored case.

I see two option 

1. create a function that take an item, and do the check and ither return the div with linkifyUrls or item[0]
2. use the ternary operator inline

Reply:
Done